setup.py

```diff
@@ -4,7 +4,7 @@
 required = [
     "Flask",
     "MarkupSafe",
-    "MySQL-python",
+    "psycopg2",
```

Comment:
it's ok to not add them in requirements.txt because we add them on heroku directly thought heroku dashboard and I think people don't need scheduler when they want to debug ouija locally(they could manually run script they want to these ouija and seta :) )